test/test_model/test_model_helpingmaterial.py

```diff
@@ -29,4 +29,4 @@ class TestModelHelpingMaterial(Test):
     def test_helpingmaterial_public_attributes(self):
         """Test public attributes works."""
         hm = HelpingMaterialFactory.create()
-        assert hm.public_attributes().sort() == hm.dictize().keys().sort()
+        assert hm.public_attributes().sort() == list(hm.dictize().keys()).sort()
```

Comment:
I think these comparisons need to be
```python
sorted(public_attributes()) == sorted(hm.dictize().keys())
```
since `.sort` sorts the list in place and returns `None`

Reply:
This is really interesting, especially because the tests are passing. I'll check with this proposed change.

Reply:
As they are written the assertions will always pass regardless of the lists contents, since they are comparing `None` to itself rather than the two sorted lists.

Reply:
Thanks. This is now fixed!